src/cmd/call_hash.rs

```diff
@@ -0,0 +1,32 @@
+use crate::cmd::{node::rpc::read_bytes, script_args::args::HexEncodedBytes};
+
+use anyhow::Result;
+use blake2::{Blake2s256, Digest};
+use clap::Parser;
+use std::path::PathBuf;
+
+/// Estimate gas for publishing modules.
```

Comment:
`/// Estimate gas for publishing modules.`
🕵️

Reply:
Done